KHLY2700/KHLY2700-IOC-01App/Db/keithley2700.db

```diff
@@ -25,22 +24,22 @@ record(bo, "$(P)DISABLE")
 record(sseq, "$(P)INIT:SEQ") {
 	field(PINI, "YES")
 	field(DESC, "Initialisation of values")
-	
+
 	# Clear all system errors
 	field(LNK1, "$(P)SYSTEM:CLEAR:SP PP")
```

Comment:
For record set points you want a consistent method for processing them. The first two records are processed inconsistently. You can remove the znam/onam and just push a 0 etc.

Reply:
Not sure what is meant by this. If you mean that sending the different STR fields is inconsistent (e.g. sending `"ON"` to `INITMODE:SP`, `"ABS"` to TIME:FORMAT:SP` etc.) each of the link fields pushes a value that the IOC/device itself would expect to see. 

Maybe I haven't understood?

Reply:
For clarification: This was a comment to suggest a consistent output method for the string sequence. In the sequence you had mixed both DO (Desired output value) and STR (Desired output string). We thought it would give a more coherent flow; I can see you've resolved this however.